plugins/CoreAdminHome/vue/src/JsTrackingCodeGenerator/JsTrackingCodeGenerator.vue

```diff
@@ -0,0 +1,614 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <ContentBlock
+    anchor="javaScriptTracking"
+    :content-title="translate('CoreAdminHome_JavaScriptTracking')"
+  >
+    <div id="js-code-options">
+      <p>
+        {{ translate('CoreAdminHome_JSTrackingIntro1') }}
+        <br/><br/>
+        {{ translate('CoreAdminHome_JSTrackingIntro2') }}
+        <span v-html="jsTrackingIntro3a"></span> {{ translate('CoreAdminHome_JSTrackingIntro3b') }}
```

Comment:
I guess the translated text of `CoreAdminHome_JSTrackingIntro3b` is not auto escaped here, causing the escaped `</head>` to be double escaped. This is also visible in the UI screenshot failure.